exe/Main.hs

```diff
@@ -83,11 +85,18 @@ main = do
 
     dir <- getCurrentDirectory
 
+    let handlers = def <>
```

Comment:
I don't think you need `def` here, because of the way this are used in `runLanguageServer`.

Reply:
Thanks, will fix